demo/public/body.css

```diff
@@ -1,7 +1,28 @@
 body {
     background-color: #f3f4fa;
-    background-image: url("media/spinner/spinner.gif");
-    background-repeat: no-repeat;
-    background-position: center center;
-    height: 200px;
-}
\ No newline at end of file
+}
+
+body::after {
+    content: "";
+    position: absolute;
+    top: 42%;
+    left: 50%;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 38%;
+    height: 1rem;
+
+    transform: translate(-50%, -42%);
+    background-color: #315ee3;
+
+    overflow: hidden;
+    text-align: center;
+    white-space: nowrap;
+    background-image: linear-gradient(45deg, hsla(0, 0%, 100%, .15) 25%, transparent 0, transparent 50%, hsla(0, 0%, 100%, .15) 0, hsla(0, 0%, 100%, .15) 75%, transparent 0, transparent);
+    background-size: 1rem 1rem;
+    animation: progress-bar-stripes 1s linear infinite;
+    transition: width .6s ease;
+    border-radius: 4px;
+}
```

Comment:
hm.. I wonder.. what are these styles for?

Reply:
@petrKavulok this is the splashscreen loader when app is not fully loaded